index.html

```diff
@@ -1 +1,58 @@
-Hello World
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Yoga</title>
+</head>
+<body>
+<h1>Yoga Poses</h1>
```

Comment:
It would be useful to insulate your HTML tags from the `body`, something like `<main>` would be appropriate.